### ISSUE TITLE
Do not find pre-trained feature_embedding files

### ISSUE THREAD

**Issue by Setftzoesto** [rutgerswiselab/DCCF#2]
I have run the main codes with parameters in the example you've given in readme.
But I can not run the code in the image, 
![bug](https://github.com/rutgerswiselab/DCCF/assets/41951946/94b4a1fe-e5b1-4cac-bd4f-fb284d608994)

it prompts me that do not find 'Electronics_paraphrase-distilroberta-base-v1.npy'.
It seems like I need to use the pre-train models to get feature_embedding？

**Comment by shuyuan-x** [comment]
Yes, you need to use the the pre-trained sentence embedding models to get feature_embedding. We applied the pre-trained paraphrase-distilroberta-base-v1 sentence embedding model in a public transformer implementation: https://github.com/UKPLab/sentence-transformers

**Comment by Setftzoesto** [comment]
OK, thank you very much!

**Comment by Setftzoesto** [comment]
My apology for disturbing.
I have read the experiment implementation details of this paper, which mentioned that (for the Amazon dataset as an example): The final feature_embedding is the average of embedding of the 'title', 'description', and 'feature'. 
I have a question about that: Is it pre_train model would finish the average job? Or I should compute the average manually?

**Comment by Setftzoesto** [comment]
Mr.Xu, apology again. Would it be possible for you to share the source code of computing feature_embedding used in your research?
I am very new to this area and try to study further. I try to program the codes but I am not sure if it's the correct method for getting feature_embedding. That would be a very great help if you want to share.

**Comment by shuyuan-x** [comment]
The feature_embedding is the average of embedding of the 'title', 'description' and 'feature'. I used pre-trained model to encode sentences separately and manually compute the average.

**Comment by Setftzoesto** [comment]
That's nice. I have achieved the function.
But there is a new problem that I need to compute IPS_EXPO_PROB.😂
大佬，能给一份代码吗，嘤嘤嘤

**Comment by shuyuan-x** [comment]
It does not need special code. We first train a IPSBiasedMF model, and then save the predicted user-item matrix as the exposure probability into IPS_EXPO_PROB file. I have updated the README to provide more details.

**Comment by Setftzoesto** [comment]
OK, thank you very much!
Oh my god, I just have found the IPSBiasedMF model in the project!😂

**Comment by shuyuan-x** [comment]
The propensity score is calculated based on the method introduced in "Unbiased recommender learning from missing-not-at-random implicit feedback". Or you can refer the equation 18 in our paper. After calculating the propensity score, save the score in propensity.npy file. Use the calculated score to train a IPSBiasedMF model.

**Comment by Setftzoesto** [comment]
OK！Very warmly instructed!
